components/automate-ui/src/app/ngrx.effects.ts

```diff
@@ -23,6 +23,7 @@ import { ScannerEffects } from './pages/+compliance/+scanner/state/scanner.effec
 import { SidebarEffects } from './services/sidebar/sidebar.effects';
 import { TeamEffects } from './entities/teams/team.effects';
 import { UserEffects } from './entities/users/user.effects';
+import { UserSelfEffects } from './entities/users/userself.effects';
```

Comment:
This file and the next, setup the new userself ngrx store logic with the global storage.